### PR TITLE
[Fix] Interview module revision with respect to core ERPNext.

### PR DESCRIPTION
**Pull request by pjamsheer**
## Feature description
- Complete revision of interview module in ONEFM with respect to core ERPNext.

## Solution description
- Deleted all the unused interview related doctype and custom fields
- Add questions section in `Interview Round`
Screenshot 2022-03-14 at 10 09 11 PM

- Populate interview question in `Interview Feedback`  from the `Interview Round` and adjusted the calculation for average rating.
Screenshot 2022-03-14 at 10 12 31 PM

- On creation of `Career History`, It  will create a `Interview` and `Interview Round`. On update and submit of the `Career History`, the `Interview` and `Interview Round` will be updated and submitted.
Screenshot 2022-03-14 at 10 17 23 PM
Screenshot 2022-03-14 at 10 16 53 PM

- In `Career History` added new configuration for `Calculate Promotions and Experience Automatically`
Screenshot 2022-03-14 at 10 50 56 AM

## Areas affected and ensured
- Removed almost all custom field and interview related doctype in One Fm module and Hiring module

## Is there any existing behavior change of other features due to this code change?
- Interview module in ONE FM
- Now it is simple and relevant

## Was this feature tested on all the browsers?
  - [x] Chrome
  - [x] Safari

### CODE REVIEW

File: one_fm/patches.txt [hunk 1]
```diff
@@ -2,4 +2,6 @@
 one_fm.patches.v1_0.change_title_in_grd #2021-07-11
 one_fm.patches.v1_0.delete_fields #2021-07-11
 one_fm.patches.v1_0.assign_roles #2021-09-13
-one_fm.patches.v1_0.update_notification_logs #2021-12-15
\ No newline at end of file
+one_fm.patches.v1_0.update_notification_logs #2021-12-15
+one_fm.patches.v1_0.delete_interview_doctype
```
Comment (Ehtesham599) [inline one_fm/patches.txt:6]: Add dates as ID to these patches so they get executed exactly once.

Comment (pjamsheer) [reply one_fm/patches.txt:6]: Patch will be executed only once

Comment (Ehtesham599) [reply one_fm/patches.txt:6]: Alright

**Review by Ehtesham599 — APPROVED** [review]
LGTM

**Pull request merged**